root/static/scripts/work.js

```diff
@@ -1,6 +1,14 @@
+// @flow
+// Copyright (C) 2015-2018 MetaBrainz Foundation
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const $ = require('jquery');
```

Comment:
Why do we need jQuery?

Reply:
It's used down here https://github.com/mwiencek/musicbrainz-server/blob/588282c88ea40f945071485367545b0eda2dcfd0/root/static/scripts/work.js#L249-L258 so this just adds the import for it so that Flow doesn't complain. (It's also a global variable, hence why it previously worked.)

The edit pages haven't really been ported to React yet, so you'll see a lot of jQuery in places.